<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flugbuch</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/logo.svg</normaloff>:/icons/logo.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QSplitter" name="vSplitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QTabWidget" name="tabWidget">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>250</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="flightsTab">
         <attribute name="title">
          <string>Flüge</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QToolButton" name="newFlightButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../resources.qrc">
                <normaloff>:/icons/new.svg</normaloff>:/icons/new.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="editFlightButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../resources.qrc">
                <normaloff>:/icons/edit.svg</normaloff>:/icons/edit.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="removeFlightButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../resources.qrc">
                <normaloff>:/icons/remove.svg</normaloff>:/icons/remove.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListView" name="flightList">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>20</horstretch>
              <verstretch>60</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="locationsTab">
         <attribute name="title">
          <string>Gelände</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QToolButton" name="newLocationButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../resources.qrc">
                <normaloff>:/icons/new.svg</normaloff>:/icons/new.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="editLocationButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../resources.qrc">
                <normaloff>:/icons/edit.svg</normaloff>:/icons/edit.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="removeLocationButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../resources.qrc">
                <normaloff>:/icons/remove.svg</normaloff>:/icons/remove.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListView" name="locationList"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="planesTab">
         <attribute name="title">
          <string>Geräte</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QToolButton" name="newPlaneButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../resources.qrc">
                <normaloff>:/icons/new.svg</normaloff>:/icons/new.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="editPlaneButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../resources.qrc">
                <normaloff>:/icons/edit.svg</normaloff>:/icons/edit.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="removePlaneButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../resources.qrc">
                <normaloff>:/icons/remove.svg</normaloff>:/icons/remove.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListView" name="planeList"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="infoWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>20</horstretch>
          <verstretch>40</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
      <widget class="QQuickWidget" name="mapWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>80</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="resizeMode">
        <enum>QQuickWidget::SizeRootObjectToView</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>22</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuKarte">
    <property name="title">
     <string>Karte</string>
    </property>
    <addaction name="actionMapOsm"/>
    <addaction name="actionMapEsri"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuKarte"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Öffnen</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Speichern unter</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>Neu</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionMapOsm">
   <property name="text">
    <string>Open Street Map</string>
   </property>
  </action>
  <action name="actionMapEsri">
   <property name="text">
    <string>Esri</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionBeenden</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
